bot.py

```diff
@@ -1232,6 +1235,22 @@ def check(answer):
     exit(0)
     return
 
+@bot.command(aliases=["image"])
+@commands.guild_only()
+async def i(ctx, *, query):
+    '''Finds an image using Google Images (safesearch off)'''
+    api_key = config["api"]["google"]
+    url = ('https://www.googleapis.com/customsearch/v1?cx=012763604623577894851:r8w2tzy60qx'
+            '&fields=items(title,link,snippet)&safe=off&nfpr=1&searchType=image')
```

Comment:
The safe search off option is a bit of a problem.  One solution may be to restrict this to NSFW channels.